### DISCUSSION TITLE
Specifying a versioning scheme when create a new release

### DISCUSSION THREAD

**Discussion by jorgerasillo** [goreleaser/goreleaser#3392] [Q&A] [answered]
We recently introduced goreleaser to our project which previously had a versioning scheme that looks like the following tag:
`2022-09-13-2111_0522a57`, i.e. YYYY-MM-DD-HHMM_git-commit. Unfortunately this meant that the only way to switch to a more strict semantic versioning scheme was to start the new versioning scheme at a version greater than `2022`, for example at: v3000.0.0 which is not great. 

Reading the [homebrew docs](https://docs.brew.sh/Formula-Cookbook#version-scheme-changes) , it seems possible to accomplish the previous example by using the `version_scheme` field. I couldn't find anything in the docs that resembled that functionality, is that possible or is there another alternative that would be possible to achieve changing the versioning scheme?

**Comment by caarlos0** [comment, marked as answer]
unfortunately no, some other things break on non-semver, so to simplify stuff a bit, goreleaser only works with semver-compatible tags.

But, you could do something like `2022.09.20` for example, which would be semver compatible I believe. That should fix your issue, and you won't need to change your version scheme that much...
